server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/SelenideConfig.java

```diff
@@ -42,7 +42,7 @@
 
     static Browser of(String s) {
       try {
-        return Browser.valueOf(s);
+        return Browser.valueOf(s.toUpperCase());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define the locale to be used in this String operation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1449)